### DISCUSSION TITLE
Where should GSD documentation be kept?

### DISCUSSION THREAD

**Discussion by joshbuker** [cloudsecurityalliance/gsd-tools#147] [General]
We have a couple of options for where to store various documentation (e.g., Policy documents #3):

- GitHub Wiki(s) (https://github.com/cloudsecurityalliance/gsd-tools/wiki)
- gsd.id website (https://github.com/cloudsecurityalliance/gsd-tools/tree/main/website)
- README.md files within gsd-tools (https://github.com/cloudsecurityalliance/gsd-tools/blob/main/CONTRIBUTING.md)
- Some other option?

What would be the easiest to use for folks and provide the best experience for those coming into the project?

Personally, I feel that using the gsd.id website and PRs to update would be a good balance between ease of updating and usability.

**Comment by kurtseifried** [comment]
So we definitely need the README.MD for top level why this exists/links to more stuff.

I imagine API docs will live at or be linked from the front page of api.gsd.id

As for the rest of it, that really depends on what documentation we end up with. I could see us using the Wiki, or perhaps pages on gsd.id, I would say we worry about this when we have some docs. Using the wiki is probably a good place to start.
